contrib/os-services/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Disable service firewalld (RedHat)
+  systemd:
+    name: firewalld
+    state: stopped
+    enabled: no
+  when:
+    - disable_service_firewall
+    - "ansible_os_family == 'RedHat'"
+
+- name: Disable service ufw (Debian)
+  systemd:
+    name: ufw
+    state: stopped
+    enabled: no
+  when:
+    - disable_service_firewall
+    - "ansible_os_family == 'Debian'"
```

Comment:
Debian can have ufw or firewalld, so better list existing systemd services and disable firewalld or ufw if present

Reply:
I see, let me add the check.